bool resultVision
---
float64 x
float64 y
bool z
bool isActive